provider/provider.go

```diff
@@ -2223,6 +2246,14 @@ func resourceShorelineObjectReadSingleAttr(name string, typ string, key string,
 			}
 		} else {
 			val = GetNestedValueOrDefault(record, ToKeyPath("attributes."+key), nil)
+			if typ == "dashboard" && key == "dashboard_type" {
+				switch val {
+				case "0":
+					val = "TAGS_SEQUENCE"
+				default:
+					val = "TAGS_SEQUENCE"
+				}
+			}
```

Comment:
Why do we need this? We should return from the API the right value. This should be fixed in our Shoreline backend.

Reply:
From what I've seen all the symbols are listed in terraform and the corresponding one is saved in `record`. See lines [2355-2364](https://github.com/shorelinesoftware/terraform-provider-shoreline/blob/main/provider/provider.go#L2355). So in `record` we save the dashboard as it appears in the db.
Later, when updating the TF state, `dashboard_type` is read from `record` where `dashboard_type` is `0`, hence the mapping